src/components/internal/OverlayTrigger.tsx

```diff
@@ -58,6 +60,8 @@ export function OverlayTrigger(props: OverlayTriggerProps) {
     isOpen: state.isOpen,
     onClose: state.close,
     placement: (placement ? `bottom ${placement}` : "bottom left") as Placement,
+    shouldUpdatePosition: showActiveBorder || undefined,
```

Comment:
**suggestion:** This shouldn't be necessary. By default `shouldUpdatePosition` is `true`. See https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/overlays/src/useOverlayPosition.ts#L84